Created a Landing Page of the entire website with proper and responsive css

r7projects-shayan opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to project-guidance here: https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes i have read it completely

### Description

Created an entire landing page for the repository with all the major resources and a navbar and a footer

Added smooth scroll and hover effects

Added responsive css effects

Fixed the watchers section

Fixes #992 

### Checklist



- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

## Screenshots

![Screenshot 2024-05-21 022834](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/7a5fbfc9-c014-45dc-9006-48564c2ae752)
![Screenshot 2024-05-21 022849](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/1d890d47-b404-4e34-aa30-2c3c3c09d003)
![Screenshot 2024-05-21 022859](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/a1ccfb1a-890e-4f5c-8c2d-0b665fe63596)
![Screenshot 2024-05-21 022908](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/d8484268-5669-4e75-8152-a622f15f78d3)
![Screenshot 2024-05-21 022915](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/9e0269fb-75bb-4fe4-8c7d-e6d68e0388a0)
![Screenshot 2024-05-21 022930](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/da664c50-6051-44bb-9bd4-071ffc272cd5)
![Screenshot 2024-05-21 022939](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/d39255f1-6abe-4b2e-b322-4e53db9cff90)
![Screenshot 2024-05-21 022950](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/b6201a74-823c-4972-b169-ab5f8d4ce15f)

![Screenshot 2024-05-27 144010](https://github.com/Kushal997-das/Project-Guidance/assets/167475552/b8a810a6-52cf-41f5-86d2-12dd80a9a72d)

## Deployment Link

https://project-guidance.vercel.app/

Kushal997-das reviewed (changes requested):
@r7projects-shayan Good work. I'll add it under `level3` if you do responsive design for all devices, including desktop, tablet, and mobile.

r7projects-shayan commented:
Alright I'll raise PR soon

On Thu, 30 May 2024, 9:04 pm Kushal Das, ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> @r7projects-shayan <https://github.com/r7projects-shayan> Good work. I'll
> add it under level3 if you do responsive design for all devices,
> including desktop, tablet, and mobile.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kushal997-das/Project-Guidance/pull/1103#pullrequestreview-2088646009>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BH5XSYALNQB4UZGBYAT62ETZE5BIJAVCNFSM6AAAAABIKZWHGCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAOBYGY2DMMBQHE>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

r7projects-shayan commented:
@Kushal997-das The work is done, I have fixed the entire website to be responsive for all devices and dimensions and also added interactive particle effects to make it look good. 

I will close this PR and create a new PR.
